server/src/test/java/org/opensearch/action/admin/cluster/remotestore/stats/RemoteStoreStatsTestHelper.java

```diff
@@ -114,7 +114,7 @@ static ShardRouting createShardRouting(ShardId shardId, boolean isPrimary) {
 
     static void compareStatsResponse(
         Map<String, Object> statsObject,
-        RemoteSegmentTransferTracker.Stats statsTracker,
+        RemoteSegmentTransferTracker.Stats segmentStatsTracker,
```

Comment:
nit: `segmentStatsTracker` -> `segmentTransferStats`

Reply:
Thanks @ashking94! I will add this in the next incremental PR

Reply:
Done in #9263